pkg/cmd/roachtest/tests/multitenant_upgrade.go

```diff
@@ -156,13 +157,10 @@ func runMultitenantUpgrade(ctx context.Context, t test.Test, c cluster.Cluster)
 		wg.Add(len(tenants))
 
 		for _, tenant := range tenants {
-			h.Background(
-				fmt.Sprintf("%s: %s", desc, tenant.name),
-				func(ctx context.Context, l *logger.Logger) error {
-					defer wg.Done()
-					return fn(ctx, l, tenant)
-				},
-			)
+			h.Go(func(ctx context.Context, l *logger.Logger) error {
```

Comment:
Nice! I think I like this more than before, where `Background` referred to either a specific stage and also the goroutine wrapper provided.